book/basics/units_basics.nim

```diff
@@ -0,0 +1,150 @@
+import nimib, nimibook
+
+nbInit()
+nbUseNimibook
+nbDoc.useLatex
+
+nbText: """
+# Using and checking units with [Unchained](https://github.com/SciNim/Unchained)
+
+Units are crucial in physics, without them we wouldn't have any consistent way of comparing two measurements. 
+You wouldn't solve an equation without checking units afterwards as a safety-check. But when it comes to coding 
+physical formulas units are often ignored or in best cases in a comment somewhere. This can be solved by using
+unit libraries like `Unchained`. What puts `Unchained` apart from many other libraries is that it does the majority
+of the work at compile-time so that you know that if the code compiles, then the units are correct. 
+Other libraries would probably check this at runtime, and it will complain first when it sees the piece of code it wants to check
```

Comment:
Maybe add another line break here to split the paragraph into 2 for easier reading.


Also, let's not be so vague:
```suggestion
Many other libraries check this at runtime, and they only complain when they run the piece of code they want to check
```